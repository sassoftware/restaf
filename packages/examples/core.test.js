let testFunctions = require('./testFunctions');
let testInfo;
beforeAll(async () => {
	try {
		testInfo = await require('./lib/setupAll')();
	} catch (err) {
		console.log(err);
		process.exit(1);
	}
});
test('identities service', async () => {
	let l = await testFunctions.identities(testInfo);
	expect(l).toMatchSnapshot();
});
