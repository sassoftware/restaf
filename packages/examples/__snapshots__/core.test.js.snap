// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`identities service 1`] = `"done"`;
